lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,17 @@
+def create_commits!(repository)
+  repository.git.commits { |c| repository.commit_for!(c.oid) }
+end
+
+def attach_commit!(ontology_version)
+  commit = Commit.find_by_commit_oid(ontology_version.commit_oid)
```

Comment:
I know, it's very improbable, but how about
```ruby
  commit = ontology_version.repository.commits.
    find_by_commit_oid(ontology_version.commit_oid)
```
with `repository.commits` meaning the association to the commit model and not the commits-iterator.